yetanotherco/aligned_layer
# refactor: batcher create a shared crate between the cli and the batcher

# Description

- Batcher now has 3 different crates:
  - `aligned-batcher` (what was called **batcher** before).
  - `aligned` (what was called **client** before).
  - `aligned-batcher-lib` (**commons** between batcher and client).

# Results

- Now `aligned` builds in ~15s without `--release` since it doesn't depend anymore on building the batcher.

# How to test

- First run `mv batcher/.env batcher/aligned-batcher/.env` to move the .env file to the corresponding directory.
- To test make sure every batcher command present in the makefile works as expected.
